homework/inner-product/arithmeticAverage.cpp

```diff
@@ -0,0 +1,21 @@
+#include "arithmeticAverage.hpp"
+
+#include <algorithm>
+#include <cmath>
+#include <functional>
+#include <numeric>
+
+double ArithmeticAverage(const std::vector<int>& first, const std::vector<int>& second) {
+    double firstSum =  std::accumulate(first.begin(), first.end(), 0, std::plus<>{}); 
+    double secondSum = std::accumulate(second.begin(), second.end(), 0, std::plus<>{});
+    return std::plus<>{}(firstSum, secondSum) / (first.size() + second.size());
```

Comment:
You can also use inner porduct instead of 2 accumulate. Try to think how you can add 2 vectors using inner product :)